modules/nw-dual-stack-convert.adoc

```diff
@@ -2,7 +2,9 @@
 [id="nw-dual-stack-convert_{context}"]
 = Converting to a dual-stack cluster network
 
-As a cluster administrator, you can convert your single-stack cluster network to a dual-stack cluster network. After converting to a dual-stack network, new and existing pods have dual-stack networking enabled.
+As a cluster administrator, you can convert your single-stack cluster network to a dual-stack cluster network. 
+
+After converting to a IPv4/IPv6 dual-stack network, new and existing pods have IPv4/IPv6 dual-stack networking enabled. If you convert from a single-stack cluster network to a IPv6/IPv4 dual-stack cluster, only new pods have IPv6/IPv4 dual-stack networking enabled
 
```

Comment:
I suggest rollback the original to 

```
After converting to dual-stack networking only newly created pods are assigned IPv6 addresses. Any pods created before the conversion must be recreated to receive an IPv6 address.
```
from https://github.com/openshift/openshift-docs/pull/75232/files

Reply:
Thanks, @zhaozhanqi . Updated but with some minor grammar changes.